pages/events/summer-school.vue

```diff
@@ -30,68 +26,78 @@
         </p>
         <p>See you next year!</p>
       </template>
-      <template slot="card">
-        <EventCard v-bind="headerData.card" vertical-layout>
+      <template #card>
+        <EventsItemCard v-bind="headerData.card" vertical-layout>
           {{ headerData.card.description }}
-        </EventCard>
+        </EventsItemCard>
       </template>
-    </AppPageHeaderWithCard> -->
+    </UiAppPageHeaderWithCard>
 
     <div class="cds--grid summer-school-page__content">
-      <!-- <AppMosaicSection
+      <UiAppMosaicSection
         class="summer-school-page__section"
         :title="mosaicData.title"
         :elements="mosaicData.tiles"
-      /> -->
+      />
 
       <section class="summer-school-page__section">
-        <!-- <h2 v-text="agendaData.title" /> -->
-        <!-- <p v-text="agendaData.subtitle" /> -->
-        <!-- <cv-tabs>
-          <cv-tab
+        <h2 v-text="agendaData.title" />
+        <p v-text="agendaData.subtitle" />
+        <bx-tabs trigger-content="Select an item" value="Week 1">
+          <bx-tab
             v-for="week in agendaData.weeks"
             :key="week.tabName"
-            :label="week.tabName"
+            :target="week.tabName"
+            :value="week.tabName"
           >
-            <AppDataTable
-              class="summer-school-page__section"
-              :columns="agendaColumnsDataTable"
-            >
-              <cv-data-table-row
+            {{ week.tabName }}
+          </bx-tab>
+        </bx-tabs>
+        <div class="summer-school-page__agenda">
+          <div
+            v-for="week in agendaData.weeks"
+            :id="week.tabName"
+            :key="week.tabName"
+            role="tabpanel"
+            :aria-labelledby="week.tabName"
+            hidden
+          >
+            <UiAppDataTable :columns="agendaColumnsDataTable">
```

Comment:
One thing I'm noticing is that on smaller viewports, I'm unable to scroll right to view the rest of the table:

We might have to do a small css adjustment for this... 

https://user-images.githubusercontent.com/6276074/223618843-b0993d7f-70e8-4c99-b79c-e6d437456470.mov

Reply:
Done in https://github.com/Qiskit/qiskit.org/pull/3017/commits/7f3e1eaec186f200b40514a9c3fa5c1021925e92